### ISSUE TITLE
Select CSS/JavaScript Frameworks

### ISSUE THREAD

**Issue by lcabrini** [Aluguntugui/aluguntugui#2]
I'm keeping this as a single issue, since there are frameworks that do both CSS and JavaScript. We need to find something that we can work with in the long run. I'm tempted to suggest to use Bulma and Vue.js since these seem to work very well together. However, I'd like to look at alternatives as well.

If we can also find django modules that help work with whatever we choose, that may also be a plus. For instance, Crispy Forms helps to create good looking forms without having to write too much HTML, which is a good thing. However, AFAIK Crispy Froms only works with Bootstrap, so if we don't want to work with that, Crispy Forms is out of the question.

**Comment by lcabrini** [comment]
I found Tailwind CSS (https://tailwindcss.com/), which seems quite interesting. One common complaint about both Bootstrap and Bulma is that all sites that use one of these frameworks tend to look more or less the same. Tailwind appears to give us more freedom, possibly at the cost of being more complex.

**Comment by lcabrini** [comment]
I've been thinking about this for a while and here is what I'm going to suggest. We stick with Bulma as the CSS framework and don't commit ourselves to a JavaScript framework at this point. My primary reeason is that I know Bulma quite well. We need to get started with the implementation and at the moment I can't think of anything that we really need Javascript for.

Eventually we might separate aluguntugui into a front-end and back-end application. But for now, I think it is more important to move on to implementation.

**Comment by enrique-cabrini** [comment]
I agree with you. The sooner we get started, the better. If we don't have an immediate use for javascript best skip it for now
